cryptol-saw-core/src/Verifier/SAW/Cryptol.hs

```diff
@@ -1108,6 +1125,46 @@ cryptolURI (p:ps) (Just uniq) =
        , uriFragment = Just frag
        }
 
+-- | Tests if the given `NameInfo` represents a name imported
```

Comment:
While I'm nitpicking about haddock markup, the standard haddock way of linking to identifiers is single quotes `'NameInfo'` instead of backticks